plugins/module_utils/vmware.py

```diff
@@ -279,17 +285,6 @@ def get_dvs_portgroup(self, portgroup):
         """
         return self.get_objs_by_name_or_moid([vim.dvs.DistributedVirtualPortgroup], portgroup)
 
-    def get_datacenter_detailed(self, name):
-        """
-        Get detailed information about a datacenter
-        Args:
-            name: The name or the ID of the datacenter
-        Returns:
-            The datacenter object
-
-        """
-        return self.get_objs_by_name_or_moid([vim.Datacenter], name)
-
```

Comment:
It looks like this function / method hasn't been released yet. So removing it wouldn't be a breaking change :partying_face: 

Could someone please double-check just to make sure?

Reply:
Yep, it's fine removing it